css/style.css

```diff
@@ -16,4 +19,144 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/*Header*/
+
+header {
+  display: flex;
+  padding: 0 5rem;
+  justify-content: space-between;
+}
+
+/*Karma Logo*/
+
+header img {
+  height: 3rem;
+  width: 3rem;
+  margin-top: 10px;
+}
+
+/*Navigation*/
+
+ul {
+  display: flex;
+  list-style: none;
+  justify-content: flex-end;
+}
+
+li {
+  padding: 0.5rem;
+}
+
+a {
+  color: gray;
+  text-decoration: none;
+}
+
+a:hover {
+  color: var(--orange-karma);
+}
+
+/*Intro Image*/
+
+.intro {
+  display: flex;
+  background-image: url("../img/first-background.jpg");
+  padding: 15.5rem 0;
+  color: white;
+  flex-direction: column;
+  align-items: center;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+  margin-bottom: 6rem;
+}
+
+/*Intro Text*/
+
+h1,
+h2,
+h3,
+.intro p {
+  font-weight: lighter;
+  text-align: center;
+}
+
+h1 {
+  font-size: 2.5rem;
+  margin: 0;
+}
+
+.intro p {
+  font-size: 1.5rem;
+}
 
+/*Intro Button*/
+
+button {
+  padding: 0.75rem 1.25rem;
+  background-color: #f26639;
+  border: none;
+  border-radius: 5px;
+  font-size: 0.75rem;
+  margin-top: 2rem;
+}
+
+/*Main*/
+
+/*Main Text*/
+
+h2 {
+  font-size: 2rem;
+}
+
+p {
+  font-size: 1.4rem;
+}
+
+/*Karma Icon Style*/
+
+.karma_icon_style {
+  display: flex;
+  justify-content: space-evenly;
+  margin-right: 3rem;
+}
+
+.karma_icon_direction {
+  display: flex;
+  flex-direction: column;
+}
+
+.karma_icon_images {
+  height: 50%;
+}
+
+hr {
+  width: 90%;
+  opacity: 25%;
+}
+
+/*footer*/
+
+/*Social Media Style*/
+
+.social_media {
+  display: flex;
+  justify-content: center;
+}
+
+.social_media_images {
+  display: flex;
```

Comment:
`display: flex` should only be used on the parent element, not the items. Given you're not styling the items (`img`s) anywhere, there's only one `img` per `.social_media_images`, and there's no other flex properties used in the parent css, I assume this is a bug?

Reply:
The unnecessary display: flex has been removed.